models/classes/TestCategoryPreset.php

```diff
@@ -125,6 +134,11 @@ public function getPluginId()
         return $this->pluginId;
     }
 
+    public function getFeatureFlag()
+    {
+        return isset($this->featureFlag) ? $this->featureFlag : false;
```

Comment:
No really need for `isset` here. Just return the property and make sure the return value is a string. Returning a bool can be confusing, because the initial value is a string. So, for example: `return (string) $this->featureFlag;`